### ISSUE TITLE
Combo boxes aren't accessible

### ISSUE THREAD

**Issue by lilmike** [mautic/mautic#10430]
### Mautic Version

4.0.0

### PHP version

7.4.21

### What browsers are you seeing the problem on?

Not relevant

### What happened?

I am blind, so use a screen reader to access my pc, and thus mautic. My screen reader I'm personally using i orca for linux desktop, but I believe this would happen with most/all screen readers for phones, mac, or pc. The gist of the issue is: All combo boxes (except on the installer, weirdly) are inaccessible css/js boxes instead. This meaning, that you cannot use the arrows keys, etc, to select an item, and must absolutely use the mouse to do such. There are plenty of ways around this, from the most involved scripting and styling around it, to similarly involved but better switching combo box frameworks to an accessible one. Or even using regular old combo boxes (although of course this would sacrifice features). Anyway, I'm just reporting it here because I really like mautic and using open source software, but as it stands I can't use the combo boxes so it falls flat.

Thanks!

-Michael.

P.S. A really, really good example of accessibility in web applications is WordPress, everything WordPress core is fully accessible, so this might be a good example.

### How can we reproduce this issue?

1. Turn on a screen reader of choice (and learn how to use its basics). Windows and mac have one built in, or windows has others free or non-free, and linux has orca.
2.
2. Access mautic, and make a new campaign.
3.
3. Open the editor to add actions and such.
4.
4. Try to select a contact source without clicking the mouse.
5. Note that is will be impossible or very, very difficult.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by RCheesley** [comment]
Thanks for reporting the issue @lilmike - I've flagged it up in the other issue (#10442) and we've done a bit of digging. I'm hopeful that we will be able to have someone look at this in the near future.
